internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -105,26 +106,29 @@ func (r *AddOnDefinitionResource) Schema(ctx context.Context, req resource.Schem
 				},
 			},
 			"readme_markdown_template": schema.StringAttribute{
-				Description: "The markdown template for the Add-on Definition README.",
+				Description: "The markdown template for the Add-on Definition README. Must be base 64 encoded.",
 				Optional:    true,
 			},
 			"terraform_cloud_workspace_details": &schema.SingleNestedAttribute{
-				Required:    true,
-				Description: "Terraform Cloud Workspace details",
+				Optional: true,
```

Comment:
This is accurate? I don't think this should technically be optional because isn't `terraform_project_id` required?

I also noticed that the server ends up doing nothing with the workspace name field, so we probably shouldn't even be asking for that and instead just ask for `terraform_project_id`?